Tools/TestWebKitAPI/Tests/WebKitCocoa/GPUProcess.mm

```diff
@@ -480,7 +480,7 @@ static NSInteger getPixelIndex(NSInteger x, NSInteger y, NSInteger width)
     EXPECT_EQ(webViewPID, [webView _webProcessIdentifier]);
 
     // FIXME: On iOS, video resumes after the GPU process crash but audio does not.
-#if !PLATFORM(IOS)
+#if !PLATFORM(IOS) && !PLATFORM(VISION)
```

Comment:
am I crazy or did you do !(PLATFORM(IOS) || PLATFORM(VISION)) in the other patch?

Reply:
Yeah, you do !(PLATFORM(IOS) || PLATFORM(VISION)) elsewhere in this patch, so you should switch to be consistent.

Reply:
somewhat chaotic all over the place, but sure. Not sure which to prefer :)

Reply:
I think this is the only instance across both patches, certainly the only instance in this patch, so I stand by switching this one.